src/main/kotlin/com/tealcube/minecraft/bukkit/mythicdrops/commands/CustomCreateCommand.kt

```diff
@@ -96,6 +97,12 @@ internal class CustomCreateCommand : BaseCommand() {
             mythicDropsPlugin.customItemYAML.set("$name.enchantments.$enchKey.minimum-level", it.minimumLevel)
             mythicDropsPlugin.customItemYAML.set("$name.enchantments.$enchKey.maximum-level", it.maximumLevel)
         }
+
+        if (itemMeta is LeatherArmorMeta) {
+            val color = itemMeta.color
+            mythicDropsPlugin.customItemYAML.set("$name.rgb", "${color.red},${color.green},${color.blue}")
```

Comment:
Similarly to what was mentioned above, I'd prefer to split this out into a full structure instead of a string. I'd recommend doing it like this instead:
```suggestion
            mythicDropsPlugin.customItemYAML.set("$name.rgb.red", color.red)
            mythicDropsPlugin.customItemYAML.set("$name.rgb.green", color.green)
            mythicDropsPlugin.customItemYAML.set("$name.rgb.blue", color.blue)
```